products/contao

```diff
@@ -0,0 +1,38 @@
+---
+title: Contao
+category: server-app
+changelogTemplate: "https://github.com/contao/contao/blob/__RELEASE_CYCLE__/CHANGELOG.md"
+auto:
+  - git: https://github.com/contao/contao.git
+
+releases:
```

Comment:
`releases` is missing versions 4.9 which is still supported (https://contao.org/en/release-plan.html).

Maybe all EOL LTS version should be documented too (https://en.wikipedia.org/wiki/Contao#Versions) ? @captn3m0 what do you think ?

Reply:
I added 4.9, too. I completely forgot about it, because the fact that there is two overlapping LTS versions is only now with 4.9 and 4.13 as the release schedule has been changed (LTS was extended). I am happy to add EOL LTS versions, too, if that is a requirement?

Reply:
I do not know if it's a requirement. Let's start with supported version for the time being and wait for @captn3m0's answer.

Reply:
I added :

- versions between 4.9 and 4.13,
- EOL LTS versions having support / EOL dates on https://en.wikipedia.org/wiki/Contao#Versions.

I do not think adding more EOL versions is useful.